crates/bevy_pbr/src/meshlet/instance_manager.rs

```diff
@@ -233,6 +233,7 @@ pub fn extract_meshlet_mesh_entities(
     }
 
     // Iterate over every instance
+    // TODO: Switch to change events to not upload every instance every frame.
```

Comment:
Yeah better instance handling on the CPU is a big todo. I know @tychedelia is also gonna look into caching material specialization for meshlet instances as well.